code/game/objects/items/weapons/storage/firstaid.dm

```diff
@@ -121,7 +121,7 @@
 /obj/item/weapon/storage/firstaid/tactical/New()
 	..()
 	if (empty) return
-	new /obj/item/clothing/tie/stethoscope( src )
+	new /obj/item/clothing/accessory/stethoscope( src )
 	new /obj/item/weapon/surgicaldrill(src)

```

Comment:
The Stethoscope only really tells pulse and respiration (both of which are viewable by examining someone)--my larger concern was that the stethoscope is the only reliable way of opening up the safe...meaning if the Captain decide to lock the disk in the safe...at least nukies have something to purchase to attempt to open it (amount of players with knowledge to open it though..)